lib/survey_gizmo/api/response.rb

```diff
@@ -42,7 +42,9 @@ def parsed_answers
         # Strip out "Other" answers that don't actually have the "other" text (they come back as two responses - one
         # for the "Other" option_id, and then a whole separate response for the text given as an "Other" response.
         if k =~ /\[question\((\d+)\),\s*option\((\d+)\)\]/
-          !answers.keys.any? { |key| key =~ /\[question\((#{$1})\),\s*option\("(#{$2})-other"\)\]/ }
+          if answers.keys.any? { |key| key =~ /\[question\((#{$1})\),\s*option\("(#{$2})-other"\)\]/ }
+            next false
+          end
```

Comment:
wouldn't you need
```
else 
  true
```

to get the `select` to work properly?  also not sure why you would need a `next`...